### DISCUSSION TITLE
Message "No RaspberryMatic container identified" beim Update RaspberryMatic (VM / OVA) unter Proxmox

### DISCUSSION THREAD

**Discussion by Sandworm-of-Dune** [OpenCCU/OpenCCU#2844] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

Bei Verwendung des Update Shellscripts wie in der Anleitung unter https://github.com/jens-maus/RaspberryMatic/wiki/Installation-Proxmox-VE#update-ct--lxc beschrieben erscheint lediglich die Meldung: 
"No RaspberryMatic container identified"

### Describe the behavior you expected

Erkennung des RapberryMatic Containers /VM , so dass ein Update möglich ist. 

### Steps to reproduce the issue

1. Installation einer des RaspberryMatic als virtuelle Maschine (VM/OVA- Full Virtual Machine) mit bewußt nicht neuester Version.
In diesem Fall: 3.75.7.20240601
Hier bin ich nach Anleitung von https://github.com/jens-maus/RaspberryMatic/wiki/Installation-Proxmox-VE#installation vorgegangen.
2. Nach Installation sowohl Proxmox als auch RaspberryMatic rebootet+ automatischer Start
3. RaspberryMatic (OVA- Full Virtual Machine) heruntergefahren.
4. Via einer geöffneten Proxmox Console Eingabe des folgenden Kommandos: 
wget -qO - https://raspberrymatic.de/install-proxmox.sh | bash -s update
5. Abruch mit Meldung: "No RaspberryMatic container identified"

![grafik](https://github.com/user-attachments/assets/2f901ec6-8e8b-4a06-b1fd-0f4670c77e6a)

### What is the version this bug report is based on?

3.75.7.20240601

### Which base platform are you running?

generic-x86_64 (Generic UEFI capable system, AMD64/x86_64)

### Which HomeMatic/homematicIP radio module are you using?

HmIP-RFUSB

### Anything in the logs that might be useful for us?

```txt
Ich habe RaspberryMatic, wie in der Anleitung empfohlen, als virtuelle Maschine (VM/OVA- Full Virtual Machine) angelegt. Es ist keine CT/LXC

Funktioniert der Updateparameter ausschließlich bei CT / LXC?

"pct list" liefert mir gar keine Einträge.

Danke schon mal. Viele Grüße
```

**Comment by jens-maus** [comment, marked as answer]
Der Punkt ist, du hast doch RaspberryMatic als VM/OVA Version installiert und nicht als LXC/CT. Und natürlich kannst du mit dem LXC update script so deine VM/OVA nicht updaten weil du eben keine LXC-Version von RaspberryMatic installiert hast - deshalb antwortet dir der Skript ja auch vollkommen richtig! Und du brauchst ja auch keine LXC-Version von RaspberryMatic, denn du hast es ja als VM/OVA in Betrieb. Folglich einfach das Update **innerhalb** der VM machen wie eine ganz normale RaspberryMatic Installation auf einem RaspberryPi oder ähnlich. Du hast hier also einfach Dinge durcheinander gebracht. 

Note: Da das kein Bug ist, sondern eine falsche Annahme deinerseits, hier mal in den Diskussionsteil verschoben. Bitte das nächste mal im Diskussionsteil erst dein Problem schildern und nicht davon ausgehen das das ein Bug ist.

**Reply by Sandworm-of-Dune** [reply]
Hallo Jens,
danke für die rasche Antwort und die Klarstellung.
Vielleicht wäre in der Anleitung bei der Sektion "Update" noch ein Hinweis recht, dass diese nicht für VM/OVA gilt bzw. dass letztere direkt innerhalb der VM aktualisiert werden müssen.
Vg

**Reply by jens-maus** [reply]
... kann ich machen, aber wenn man Proxmox nutzt und mächtig ist, dann sollte man den Unterschied von LXC/CT vs OVA/VM aus dem ff kennen 😁

**Reply by Sandworm-of-Dune** [reply]
zu meiner Verteidigung: Ich bin noch Proxmox-Neuling. Probiere gerade noch rum, bevor ich mein System darauf komplett migriere.
Ich verstand es es ursprünglich als Alternative direkt über Proxmox dann auch die RaspberryMatic Installation upzudaten.
Verstanden geht nur als LXC.
